librz/core/cbin.c

```diff
@@ -31,6 +31,25 @@
 #define IS_MODE_NORMAL(mode)    (!(mode))
 #define IS_MODE_CLASSDUMP(mode) ((mode)&RZ_MODE_CLASSDUMP)
 
+/**
+ * \brief Store some display name from RzBinSymbol
+ * \see bin_sym_name_init
+ * \see bin_sym_name_fini
+ */
+typedef struct bin_sym_names_t {
+	const char *pfx; ///< prefix for flags
+	char *name; ///< raw symbol name
+	char *symbolname; ///< display symbol name
+	char *libname; ///< name of the lib this symbol is specific to, if any
+	char *nameflag; ///< flag name for symbol
+	char *demname; ///< demangled raw symbol name
+	char *demflag; ///< flag name for demangled symbol
+	char *classname; ///< classname
+	char *classflag; ///< flag for classname
+	char *methname; ///< methods [class]::[method]
+	char *methflag; ///< methods flag sym.[class].[method]
+} BinSymNames;
```

Comment:
What if we need to get that information from the API? Like it's currently done in https://github.com/rizinorg/rizin/blob/dev/test/integration/test_bin_symbols.c
It should not be a private structure. Maybe put it in `<rz_bin.h>` instead?

Reply:
tbh, that structure was used only for printing and nothing else.
also this structure is kinda meh.

Reply:
For printing is fine. This is very common use case to parse the binary and get information about symbols, imports, exports using the API. This feature should not be removed. If you have any suggestions how to make a better structure - please do, if not, please keep it as is.

Reply:
Sorry, I misunderstood, I used another structure, not this one. Ignore this comment.